src/main/resources/view/TestWindow.fxml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.net.URL?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.Scene?>
+<?import javafx.scene.control.Menu?>
+<?import javafx.scene.control.MenuBar?>
+<?import javafx.scene.control.MenuItem?>
+<?import javafx.scene.control.SplitPane?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<?import javafx.scene.layout.HBox?>
+<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
+         title="Bookopedia" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+    <icons>
+        <Image url="@/images/address_book_32.png" />
+    </icons>
+    <scene>
+        <Scene>
+            <stylesheets>
+                <URL value="@DarkTheme.css" />
+                <URL value="@Extensions.css" />
+            </stylesheets>
+
+            <VBox>
+                <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
+                    <Menu mnemonicParsing="false" text="File">
+                        <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
+                    </Menu>
+                    <Menu mnemonicParsing="false" text="Help">
+                        <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+                    </Menu>
+                </MenuBar>
+
+                <StackPane VBox.vgrow="ALWAYS">
+                    <HBox>
+                        <StackPane HBox.hgrow="ALWAYS">
+                            <VBox>
+                                <StackPane VBox.vgrow="ALWAYS" fx:id="viewDisplayPlaceHolder" styleClass="pane-with-border"
+                                            minHeight="300">
+                                    <padding>
+                                        <Insets top="5" right="10" bottom="5" left="10" />
+                                    </padding>
+                                </StackPane>
+                                <StackPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="pane-with-border"
+                                           minHeight="100" prefHeight="100">
+                                    <padding>
+                                        <Insets top="5" right="10" bottom="5" left="10" />
+                                    </padding>
+                                </StackPane>
+
+                                <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
+                                    <padding>
+                                        <Insets top="5" right="10" bottom="5" left="10" />
+                                    </padding>
+                                </StackPane>
```

Comment:
I like the overall redesign of the GUI particularly how the command box is at the bottom now!